public/locales/en/expectations.json

```diff
@@ -2,7 +2,7 @@
   "header-purpose": "Check the status of your passport application",
   "header-avoid-waiting": "Avoid waiting on the phone and request the status of your application online",
   "header-who-can-check": "Who can check their status online",
-  "header-privacy": "Privacy",
+  "header-privacy": "Privacy Notice Statement",
```

Comment:
This should be sentence-cased.
See: <https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-1-1>